### PR TITLE
Smoke test annotations, OpenFin test fixes

### PR DESCRIPTION
**Pull request by SHolleworth**
Added annotations to 2 tests for a minimal smoke test set.

Fixed some issues with the OpenFin tests.

Tidied up some tests and env files.

### CODE REVIEW

File: src/client/e2e/credit.spec.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { test } from "./fixtures"
 
 test.describe("Credit", () => {
   test.describe("New RFQ", () => {
-    test("When I select Googl instrument and click Send RFQ button then I should see a GOOGL RFQ created on the RFQ sections and I can accept any value", async ({
+    test("When I select Googl instrument and click Send RFQ button then I should see a GOOGL RFQ created on the RFQ sections and I can accept any value @smoke", async ({
```
Comment (algreasley) [inline src/client/e2e/credit.spec.ts:6]: _to discuss:_ should we put tags at the beginning or end .. end makes more sense as it doesn't mess up your reading of the purpose of the test, but I fear it is a little _invisible_ ..
it is a pity we do not have decorators yet, perhaps we could if we jumped on TS 5 ?

File: src/client/e2e/fixtures.ts [hunk 2]
```diff
@@ -58,18 +59,18 @@ const urlPathToCreditPage = (path: string): CreditPage => {
 }
 
 export const test = base.extend<IPlaywrightFixtures>({
-  browser: async ({}, use) => {
-    try {
+  browser: async ({}, use, workerInfo) => {
+    if (workerInfo.project.name === "openfin") {
```
Comment (algreasley) [inline src/client/e2e/fixtures.ts:63]: perhaps use a little util, so we can isolate the project ID for OpenFin runs

File: src/client/e2e/fixtures.ts [hunk 3]
```diff
@@ -125,25 +126,27 @@ export const test = base.extend<IPlaywrightFixtures>({
...
+      const runtimeConnection = await chromium.connectOverCDP(
+        `${process.env.NOTIFICATION_RUNTIME_ADDRESS}`,
       )
-    }
 
-    const pages = await contexts[0].pages()
+      const contexts = runtimeConnection.contexts()
+      if (contexts.length !== 1) {
+        throw Error(
+          `Unexepcted Context(s) for Notification: Expected 1, Found ${contexts.length}`,
```
Comment (algreasley) [inline src/client/e2e/fixtures.ts:138]: little typo (inherited) ..

File: src/client/e2e/test-data/blotter-data.csv
Comment (algreasley) [inline src/client/e2e/test-data/blotter-data.csv:1]: why do we need to check these files in?

File: src/client/public-openfin/launcher.json [hunk 4]
```diff
@@ -21,7 +21,7 @@
     }
   },
   "runtime": {
-    "version": "17.85.53.10",
+    "version": "22.94.67.8",
```
Comment (algreasley) [inline src/client/public-openfin/launcher.json:24]: don't check in please .. affects the main app, which is not tested on this version .. will get increased to V28/29 by OF upgrade very soon anyway

**Review by algreasley — COMMENTED** [review]
nice, @SHolleworth  - good bit of `await`ing tidy up - tests much cleaner now

File: src/client/e2e/test-data/blotter-data.csv
Comment (SHolleworth) [reply src/client/e2e/test-data/blotter-data.csv:1]: I did wonder if we needed to check this in, removing

File: src/client/e2e/credit.spec.ts (hunk 1 above)
Comment (SHolleworth) [reply src/client/e2e/credit.spec.ts:6]: What kind of issues do you think less visibility could create?

Comment (algreasley) [reply src/client/e2e/credit.spec.ts:7]: not important ... let's leave it and I'll hope that Decorators will be the way forward, and I can annotate the tests like I used to in the past (was Java/Selenium not JS)

**Review by algreasley — APPROVED** [review]
nice .. looking good!

**Pull request merged**